### ISSUE TITLE
Minor changes to the data: deleting occurrences that cannot be hidden

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/ASL-DAI#436]
This sign cannot be hidden, because it is the only one:

<img width="986" alt="Screen Shot 2023-04-28 at 6 48 05 PM" src="https://user-images.githubusercontent.com/13629122/235266176-39887658-9523-4ba0-9c26-87d548ddf970.png">

The same is true for this one:

<img width="981" alt="Screen Shot 2023-04-28 at 6 51 12 PM" src="https://user-images.githubusercontent.com/13629122/235266360-0915628a-98b3-4f52-8716-ed85caeb844c.png">

**Comment by cneidle** [comment]
THIS ALSO NEEDS TO BE DELETED:

<img width="1118" alt="Screen Shot 2023-05-04 at 6 43 26 PM" src="https://user-images.githubusercontent.com/13629122/236345113-7dc381c7-4ece-449c-be27-34872ab1a146.png">

**Comment by cneidle** [comment]
these also:

<img width="978" alt="Screen Shot 2023-05-10 at 9 16 30 AM" src="https://github.com/DCS-LCSR/ASL-DAI/assets/13629122/d7e4bfee-68f6-4d57-8ba9-46764232e8eb">

<img width="1017" alt="Screen Shot 2023-05-10 at 9 26 37 AM" src="https://github.com/DCS-LCSR/ASL-DAI/assets/13629122/d6e1864a-f754-4f12-a7c6-a3fdd95ab3aa">


<img width="984" alt="Screen Shot 2023-05-10 at 9 00 25 AM" src="https://github.com/DCS-LCSR/ASL-DAI/assets/13629122/7314c207-1dc6-4100-acbe-435e510a736d">


<img width="1087" alt="Screen Shot 2023-05-07 at 2 54 30 PM" src="https://user-images.githubusercontent.com/13629122/236697148-fc7506e2-d820-4566-b788-7fa369bc08bd.png">


<img width="1054" alt="Screen Shot 2023-05-04 at 6 59 44 PM" src="https://user-images.githubusercontent.com/13629122/236347837-d08fdfd2-ff2b-4e3f-b49e-74a23b24c2e5.png">

<img width="1080" alt="Screen Shot 2023-05-06 at 7 02 09 PM" src="https://user-images.githubusercontent.com/13629122/236649712-b7067386-4b68-4223-a162-ab1c27e58bd8.png">

**Comment by cneidle** [comment]
<img width="1161" alt="Screen Shot 2023-05-06 at 3 03 17 PM" src="https://user-images.githubusercontent.com/13629122/236642207-89661aea-dbc4-4746-aadb-9c2fadeb4f01.png">

The compound parts will also have to be removed for this:.  (5)HEART* and 
*KIND/GENEROUS ((5)HEART+KIND/GENEROUS)

**Comment by cneidle** [comment]
done

**Comment by cneidle** [comment]
Previously we had classified "pshaw" as a gesture.  This was changed in the latest annotations to a lexical sign:  PSHAW.

But it is still being classified as a gesture.  This nedes to be fixed.   The gesture  "pshaw" needs to be deleted and PSHAW needs to be classified as lexical.

<img width="764" alt="Screen Shot 2023-05-06 at 4 02 33 PM" src="https://user-images.githubusercontent.com/13629122/236644482-aa574028-85d7-4401-a564-83e8a49ec2c8.png">



<img width="922" alt="Screen Shot 2023-05-06 at 4 02 22 PM" src="https://user-images.githubusercontent.com/13629122/236644502-21d6af8d-07dd-4263-9669-f81d4d964616.png">
